locales/en.default.schema.json

```diff
@@ -159,6 +159,21 @@
         }
       }
     },
+    "logo": {
+      "name": "Logo",
+      "settings": {
+        "logo_image": {
+          "label": "Logo"
+        },
+        "logo_width": {
+          "label": "Custom logo width"
```

Comment:
I wonder about this label because in some scenarios the width is smaller than what we have in the setting. In this case you have 250 in the setting and 193 in the actual size. thoughts? 👀 
<img width="1198" alt="Screen Shot 2022-11-10 at 1 13 06 PM" src="https://user-images.githubusercontent.com/19962891/201174298-0805a862-ea77-4832-87c2-f904c8664425.png">

Reply:
Yea, I think it is a more accurate description so we should change it.